### PR TITLE
Fix for skipping poll for active projects

### PR DESCRIPTION
**Pull request by itavero**
As mentioned in PR #24, we still experienced some issues with the previous change.
I've tested this on our environment and this does work without any problems.

### CODE REVIEW

File: src/main/java/hudson/plugins/accurev/AccurevSCM.java [hunk 1]
```diff
@@ -402,13 +404,13 @@ public PollingResult compareRemoteRevisionWith(@Nonnull Job<?,?> project, @Nulla
                                                    @Nullable FilePath workspace, @Nonnull TaskListener listener,
                                                    @Nonnull SCMRevisionState baseline) throws IOException, InterruptedException {
 //        TODO: Implement SCMRevisionState?
-		if (activeProject == null) {												   
-            activeProject = project;
-        } else {
+		if (activeProject != null && activeProject.isBuilding()) {
```
Comment (casz) [inline src/main/java/hudson/plugins/accurev/AccurevSCM.java]: can you fix the indent :)

**Comment by casz** [comment]
Did you fix it polling only on master?

**Comment by itavero** [comment]
Yes, it works now.
Apparently my fix did not fix enough. ;)
It still reported not needing a workspace.
After I added some more logging, I finally found that this was the case.

It looks to me that the conditions that were combined with OR operators, where not correctly combined.
That's why I split them, so I could clearly see the decisions being made.

File: src/main/java/hudson/plugins/accurev/AccurevSCM.java (hunk 1 above)
Comment (itavero) [inline src/main/java/hudson/plugins/accurev/AccurevSCM.java]: You are commenting on an old version I believe.

**Pull request merged**